src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,12 @@
+$white-color: #fff;
+$border-color: #f3f3f3;
+$accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$font-size-12px: 12px;
+$font-size-10px: 10px;
```

Comment:
Not the best names of variables. 12px is always 12px. Better to use smth like $text-small, or mixin with arguments for font style.